tests/format/yaml/block-literal/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -45,6 +45,43 @@ printWidth: 80
 ================================================================================
 `;
 
+exports[`ideographic-space.yml - {"proseWrap":"always"} format 1`] = `
+====================================options=====================================
+parsers: ["yaml"]
+printWidth: 80
+proseWrap: "always"
+                                                                                | printWidth
+=====================================input======================================
+a: |
+  　x
+  　y
+
+=====================================output=====================================
+a: |
+  　x
+  　y
+
+================================================================================
+`;
+
+exports[`ideographic-space.yml format 1`] = `
+====================================options=====================================
+parsers: ["yaml"]
+printWidth: 80
+                                                                                | printWidth
+=====================================input======================================
+a: |
+  　x
+  　y
+
+=====================================output=====================================
+a: |
+  　x
+  　y
```

Comment:
Whitespace is hard to review, and it can easily accidentally changed, can use `snippets` to test? Example: https://github.com/prettier/prettier/blob/ece93681f1010796e7d8eb4394196ccaef0cbc9c/tests/format/jsx/whitespace/jsfmt.spec.js#L4

Reply:
Thanks for your advice 😄  I'll change the input block like this!